core/src/main/scala/caliban/validation/Validator.scala

```diff
@@ -52,12 +52,17 @@ object Validator {
    * Verifies that the given schema is valid. Fails with a [[caliban.CalibanError.ValidationError]] otherwise.
    */
   def validateSchema[R](schema: RootSchemaBuilder[R]): IO[ValidationError, RootSchema[R]] = {
```

Comment:
this seems to be unused now, so we could mark it as `@deprecated`

Reply:
I would keep that, just in case it's useful for some